bundle-workflow/tests/test_integ_workflow/integ_test/test_integ_test_suite.py

```diff
@@ -58,13 +58,7 @@ def test_execute_with_multiple_test_configs(
                     "/tmpdir/job-scheduler",
                     True,
                     False,
-                ),
-                call(
-                    "integtest.sh -b localhost -p 9200 -s false -v 1.1.0",
-                    "/tmpdir/job-scheduler",
-                    True,
-                    False,
-                ),
+                )
```

Comment:
This is incorrect. This test is to check all test-configs are executed as part of a component test suite. You are removing check for without-security config run here.

Reply:
Addressed in https://github.com/opensearch-project/opensearch-build/pull/630